deployments/kubernetes/chart/gitwebhookproxy/values.yaml

```diff
@@ -33,6 +33,12 @@ gitWebhookProxy:
         port: 80
         protocol: TCP
         targetPort: 8080
+  securityContext:
```

Comment:
Have you tested with these constraints? Does it work fine?

Reply:
yes it does